src/main/java/net/kodehawa/mantarobot/core/listeners/entities/CachedMessage.java

```diff
@@ -29,7 +29,7 @@ public CachedMessage(long author, String content) {
     }
 
     public User getAuthor() {
-        return MantaroBot.getInstance().getShardManager().getUserById(author);
+        return MantaroBot.getInstance().getShardManager().retrieveUserById(author).complete();
```

Comment:
Might be worth changing this to take a Guild argument and fetch the member instead

Reply:
Done :o